app/controllers/projects_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ProjectsController < ApplicationController
   after_action :verify_authorized
   after_action :verify_policy_scoped, only: :index
+  before_action :validate_year_month, only: :openers
```

Comment:
I would like to see a seperate controller for this to stop projects controller getting bloated, might just be a preference thing though, so I will keep going, take this as non-blocking!